README.md

```diff
@@ -76,26 +76,30 @@ format specifications to be used.
 
 - C++ using STL ([Repository](https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime), [Issues](https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/issues))
 - C#/.NET ([Repository](https://github.com/kaitai-io/kaitai_struct_csharp_runtime), [Issues](https://github.com/kaitai-io/kaitai_struct_csharp_runtime/issues))
+  - [NuGet package](https://www.nuget.org/packages/KaitaiStruct.Runtime.CSharp/)
 - Go ([Repository](https://github.com/kaitai-io/kaitai_struct_go_runtime), [Issues](https://github.com/kaitai-io/kaitai_struct_go_runtime/issues))
 - JavaScript ([Repository](https://github.com/kaitai-io/kaitai_struct_javascript_runtime), [Issues](https://github.com/kaitai-io/kaitai_struct_javascript_runtime/issues))
+  - [npm package](https://www.npmjs.com/package/kaitai-struct)
   - Examples ([Repository](https://github.com/kaitai-io/kaitai_struct_examples))
   - Webpack loader for kaitai-struct .ksy definitions ([Repository](https://github.com/kaitai-io/kaitai-struct-loader), [Issues](https://github.com/kaitai-io/kaitai-struct-loader/issues))
 - Java ([Repository](https://github.com/kaitai-io/kaitai_struct_java_runtime), [Issues](https://github.com/kaitai-io/kaitai_struct_java_runtime/issues))
+  - [Java jar at maven package](https://search.maven.org/artifact/io.kaitai/kaitai-struct-runtime)
```

Comment:
I would slightly reword this.

First, JAR is an abbreviation of Java ARchive or maybe a file extension `.jar` of it, so I'd make this explicit by using `JAR` or `.jar` (we don't mean this `jar` 🏺). Similarly, Maven is a name of the artifact repository, so I'd put the capital letter there (`Maven`).

Second, I don't really understand what the word `package` relates to - is it _Java .jar package_, or is there an artifact repository called _Maven package_? If the former meaning applies, I would keep these related words adjacent (`Java .jar package at Maven`).

```suggestion
  - [Java .jar package at Maven](https://search.maven.org/artifact/io.kaitai/kaitai-struct-runtime)
```